### PR TITLE
SDIT-1930: ✨ Latest sqs and add in eventType for sent messages

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/SynchronisationQueueService.kt [hunk 1]
```diff
@@ -33,6 +34,9 @@ class SynchronisationQueueService(
       SendMessageRequest.builder()
         .queueUrl(queue.queueUrl)
         .messageBody(sqsMessage.toJson())
+        .messageAttributes(
+          mapOf("eventType" to MessageAttributeValue.builder().dataType("String").stringValue("prisoner-from-nomis-synchronisation-$messageType").build()),
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/SynchronisationQueueService.kt:38]: I'm confused as to the difference between this message and the one from the migration queue service - are they both synchronisation messages?

Comment (andymarke) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/SynchronisationQueueService.kt:38]: They will go to different queues though the scenario will be similar.
1) An error create a mapping while running a bulk migration
2) An error creating a mapping (and other retries) while processing a NOMIS xtag event

**Review by andymarke — APPROVED** [review]
lgtm

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/SynchronisationQueueService.kt (hunk 1 above)
Comment (petergphillips) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/SynchronisationQueueService.kt:38]: turns out the migration one was never used, so have removed it

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/MigrationQueueService.kt [hunk 2]
```diff
@@ -52,23 +55,6 @@ class MigrationQueueService(
     }
   }
 
-  suspend fun <T : Enum<T>> sendMessage(message: T, context: SynchronisationContext<*>) {
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/MigrationQueueService.kt:55]: this did feel weird - a migration service using a synchronisation context, but turns out it wasn't used

**Pull request merged**